snapcraft/internal/pluginhandler.py

```diff
@@ -324,23 +324,9 @@ def migratable_fileset_for(self, step):
         return _migratable_filesets(fileset, self.code.installdir)
 
     def _organize(self):
-        organize_fileset = getattr(self.code.options, 'organize', {}) or {}
+        fileset = getattr(self.code.options, 'organize', {}) or {}
```

Comment:
I would also remove `filesets`, `snap`, `prime`, `stage`, `source`, `source-.*` and `organize` out of options and into the pluginhandler itself.